src/Items/ItemFishingRod.h

```diff
@@ -89,29 +89,41 @@ class cItemFishingRodHandler :
 						return true;
 					}
 				);
+				a_Player->UseEquippedItem(5);
 			}
 			else if (FloaterInfo.CanPickup())
 			{
+				UInt32 LotSLevel = std::min(a_Player->GetEquippedItem().m_Enchantments.GetLevel(cEnchantments::enchLuckOfTheSea), 3);
```

Comment:
you need to use the unsigned literal `3u` because the lhs of `std::min` is unsigned.